htmlbasic/bootstrap2.html/homework2-1.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
+</head>
+<body>
+
+<div class="jumbotron text-center">
+  <h1>Alexa's Cafe Menu Friday</h1>
+  <p>We have a selection of American dishes to enjoy!</p>
+
```

Comment:
I am missing a navbar. The assignment asked for the conversion of all previous 4 pages